docs/resources/Guild.md

```diff
@@ -115,6 +115,7 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | VERIFIED               | guild is verified                                                               |
 | PARTNERED              | guild is partnered                                                              |
 | PUBLIC                 | guild is public                                                                 |
+| COMMUNITY              | guild is a community                                                            |
```

Comment:
This doesn't really explain anything

Reply:
The other ones don't really explain anything either, like "guild is public", so I assumed that's how Discord wants it

Reply:
At this point you might as well remove the description column and just list the feature names. This shouldn't become the standard.